tests/test_salesforce_all_fields_non_custom_rest.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test that with only non-custom fields selected for a stream automatic fields and non custom fields  are still replicated
+"""
+
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from sfbase import SFBaseTest
+
+
+class SFNonCustomFieldsTest(AllFieldsTest, SFBaseTest):
+
+    salesforce_api = 'REST'
+
+    @staticmethod
+    def name():
+        return "tt_sf_all_fields_non_custom"
+
+    def streams_to_test(self):
+        if self.partitioned_streams:
+            return self.partitioned_streams
+        return self.partition_streams(self.get_streams_with_data())
+
+    def streams_to_selected_fields(self):
+        found_catalogs = AllFieldsTest.found_catalogs
+        conn_id = AllFieldsTest.conn_id
+        non_custom_fields = self.get_non_custom_fields(found_catalogs, conn_id)
+        return non_custom_fields
+
+    def test_non_custom_fields(self):
+        for stream in self.streams_to_selected_fields():
```

Comment:
Should this be wrapped in a `subTest` so that in one run you can see all of the failing streams?